content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -85,12 +85,14 @@ Kubernetes manifests can be defined in YAML or JSON. The file extension `.yaml`,
 `.yml`, and `.json` can be used.
 
 ```bash
-kubectl apply -f ./my-manifest.yaml            # create resource(s)
-kubectl apply -f ./my1.yaml -f ./my2.yaml      # create from multiple files
-kubectl apply -f ./dir                         # create resource(s) in all manifest files in dir
-kubectl apply -f https://git.io/vPieo          # create resource(s) from url
-kubectl create deployment nginx --image=nginx  # start a single instance of nginx
-kubectl explain pods                           # get the documentation for pod manifests
+kubectl apply -f ./my-manifest.yaml                                                               # create resource(s)
+kubectl apply -f ./my1.yaml -f ./my2.yaml                                                         # create from multiple files
+kubectl apply -f ./dir                                                                            # create resource(s) in all manifest files in dir
+kubectl apply -f https://git.io/vPieo                                                             # create resource(s) from url
+kubectl create deployment nginx --image=nginx                                                     # start a single instance of nginx
+kubectl create job hello --image=busybox -- echo "Hello World"                                    # create a job which prints "hello world"
+kubectl create cronjob hello --image=busybox --schedule="*/1 * * * *" -- echo "Hello World"       # create a cron job which prints "hello world" for every minute
```

Comment:
Thanks for adding this hints. I'd suggest we put the new lines this way:

```
kubectl create job hello \                     # create a job which prints "hello world"
   --image=busybox -- echo "Hello World"                                    
```

The intent is to avoid horizontal scrolling of the code segment considering that readers may have a screen not so big.

Reply:
@tengqm - Good feedback. I have made the change. Please review